charts/ingress-nginx/templates/_params.tpl

```diff
@@ -60,9 +60,6 @@
 {{- if .Values.controller.enableTopologyAwareRouting }}
 - --enable-topology-aware-routing=true
 {{- end }}
-{{- if .Values.controller.disableLeaderElection }}
-- --disable-leader-election=true
-{{- end }}
```

Comment:
The value is still there, why remove this?

Reply:
Because it's not supported neither by v1.9.x nor v1.10.x, yet. So it basically breaks the controller on boot.

Yet another reason why I never implemented changes in the `-upstream` repo and rather applied them from case to case.